Blocks/URL/url.h

```diff
@@ -356,7 +359,7 @@ struct URL : URLParametersExtractor, URLWithoutParametersParser {
 
   // Extra parameters for previous host and port are provided in the constructor to handle redirects.
   URL(const std::string& url,
-      const std::string& previous_scheme = kDefaultScheme,
+      const std::string& previous_scheme = "",
       const std::string& previous_host = "",
       const int previous_port = 0)
```

Comment:
We've already changed `int port` to `uint16_t` port in some places - let's fix it here as well?